src/containers/App/types.js

```diff
@@ -0,0 +1,12 @@
+export type BuildInformation = {
```

Comment:
this should be removed

Reply:
These are not props, it's part of the state.

Reply:
ah yes you are right